### PR TITLE
Use ci-helpers

### PR DESCRIPTION
**Pull request by astrofrog**
cc @bsipocz @eteq

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -14,40 +14,32 @@ env:
...
         - os: linux
           env: PYTHON_VERSION=2.7 SETUPTOOLS_VERSION=dev
+               CONDA_DEPENDENCIES="mercurial mock"
 
 before_install:
 
-    # Use utf8 encoding. Should be default, but this is insurance against
-    # future changes
-    - export PYTHONIOENCODING=UTF8
+    - if [[ $TRAVIS_OS_NAME == linux ]]; then apt-get update; apt-get install graphviz; fi
```
Comment (bsipocz) [inline .travis.yml]: Is there a reason ``astropy-helpers`` is not tested on the container travis? That case ``graphviz`` could be installed there, isn't it?

Comment (astrofrog) [inline .travis.yml]: I don't think that would work for the OS X builds, but I can look into it

File: .travis.yml [hunk 2]
```diff
@@ -14,40 +14,30 @@ env:
     - PYTHON_VERSION=3.3
     - PYTHON_VERSION=3.4
   global:
-      - SETUPTOOLS_VERSION=stable
+    - SETUPTOOLS_VERSION=stable
+    - CONDA_DEPENDENCIES="pytest<2.6 sphinx cython numpy"
```
Comment (embray) [inline .travis.yml]: This is contrary to #127 where I removed the '<2.6' pin for pytest.

Comment (astrofrog) [inline .travis.yml]: Fixed

**Comment by astrofrog** [comment]
@embray @bsipocz - ready for review, now that the tests are passing!

**Comment by embray** [comment]
Ah, I missed that this is working now.  Thanks!

**Comment by bsipocz** [comment]
This was merged to master as well as 1.0.6 no matter what the release scripts say (was part of a nested  master merge, thus not picked up).

**Pull request merged**